iocage_lib/ioc_fetch.py

```diff
@@ -860,8 +864,7 @@ def fetch_update(self, cli=False, uuid=None):
                 },
                 _callback=self.callback,
                 silent=self.silent)
-
-        shutil.copy("/etc/resolv.conf", f"{mount_root}/etc/resolv.conf")
+            shutil.copy("/etc/resolv.conf", f"{mount_root}/etc/resolv.conf")
```

Comment:
why not the same here ?

Reply:
It's been a while since I opened this, so my memory is limited. Based on the code, there are these code paths:

1. fetch_release: Get a fresh release, cli=False (where copying in a fresh resolv.conf is fine, I guess)
2. jail is a basejail: cli=False - here probably the config should be respected - I never used basejails with iocage
3. everything else (cloned jails): cli=True, this is what I tested

So, in case 1. is also okay (not sure if there is a config involved at that point), yeah, I guess it could be done on all. Somebody would have to test it though :)